src/actions.rs

```diff
@@ -33,9 +33,10 @@ pub fn list_episodes(search: &str) -> Result<()> {
                 .for_each(|(num, ep)| {
                     writeln!(
                         &mut handle,
-                        "({}) {}",
+                        "({}) {} [{}]",
                         episodes.len() - num,
-                        ep.title().unwrap()
+                        ep.title().unwrap(),
+                        ep.duration().unwrap()
```

Comment:
I'm not certain that this is frequently populated, so this would need to work when not present (not use `unwrap`)